proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -53,6 +53,7 @@ enum ResourceType {
   RESOURCE_TYPE_CURATED_PLUGIN = 8;
   RESOURCE_TYPE_TOKEN = 10;
   RESOURCE_TYPE_SCIM_TOKEN = 11;
+  RESOURCE_TYPE_REPOSITORY_DRAFT_COMMIT = 12;
```

Comment:
Should we re-use `RESOURCE_TYPE_REPOSITORY_COMMIT` here?

Reply:
jynx haha